xml/cap_admin_minibroker.xml

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.minibroker"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Minibroker</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
```

Comment:
Maybe use a more descriptive title, like "Provisioning Services with Minibroker" ?

Reply:
Title updated in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68